app/models/idea.rb

```diff
@@ -0,0 +1,6 @@
+class Idea < ApplicationRecord
+    belongs_to :user
+    has_many :reviews
```

Comment:
Line 
```ruby
has_many :reviews
```
should be something like:
```ruby
has_many :reviews, dependent: :destroy
```
To gracefully destroy associated Reviews on destruction of an Idea